packages/cli/src/__tests__/e2e/build-rs.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { polywrapCli } from "./utils";
+
+import { runCLI } from "@polywrap/test-env-js";
+import { GetPathToCliTestFiles } from "@polywrap/test-cases";
+import fs from "fs";
+import path from "path";
+
+jest.setTimeout(500000);
+
+describe("e2e tests for build command", () => {
+  const testCaseRoot = path.join(GetPathToCliTestFiles(), "wasm/build-cmd/rust");
+  const testCases = fs
+    .readdirSync(testCaseRoot, { withFileTypes: true })
+    .filter((dirent) => dirent.isDirectory())
+    .map((dirent) => dirent.name);
+
+  const getTestCaseDir = (index: number) =>
+    path.join(testCaseRoot, testCases[index]);
+
+  describe("Image strategy", () => {
+    it("Builds for rust", async () => {
+      const { exitCode: code, stdout: output } = await runCLI({
+        args: ["build", "-v", "-s", "image"],
+        cwd: getTestCaseDir(0),
+        cli: polywrapCli,
+      });
+  
+      const buildDir = `./build`;
+  
+      expect(code).toEqual(0);
+      expect(output).toContain(`Artifacts written to ${buildDir}`);
+      expect(output).toContain(`WRAP manifest written in ${buildDir}/wrap.info`);
+    });
+  })
+
+  // NOTE: Skipped because CI needs system prequisites: bindgen, snip, etc.
```

Comment:
Agreed. Yeah I was thinking about including the dependencies in a Readme or an install.sh script; because it shouldn't be the Polywrap's CLI responsibility to install these permanent system dependencies on the user's machine.

Reply:
I almost feel like, we could get this merged and open an issue on where to include that in the docs, or maybe add a command flag to output an install script for the user to run. Thoughts? @dOrgJelli

Reply:
> we could get this merged and open an issue on where to include that in the docs

yah lets do it